enrollment-server/src/test/java/com/wultra/app/enrollmentserver/impl/service/MobileTokenServiceTest.java

```diff
@@ -0,0 +1,105 @@
+package com.wultra.app.enrollmentserver.impl.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+import com.wultra.app.enrollmentserver.database.OperationTemplateRepository;
+import com.wultra.app.enrollmentserver.database.entity.OperationTemplateEntity;
+import com.wultra.app.enrollmentserver.errorhandling.MobileTokenConfigurationException;
+import com.wultra.app.enrollmentserver.impl.service.converter.MobileTokenConverter;
+import com.wultra.security.powerauth.client.PowerAuthClient;
+import com.wultra.security.powerauth.client.model.error.PowerAuthClientException;
+import com.wultra.security.powerauth.client.model.request.OperationListForUserRequest;
+import com.wultra.security.powerauth.client.model.response.OperationDetailResponse;
+import com.wultra.security.powerauth.lib.mtoken.model.entity.Operation;
+import com.wultra.security.powerauth.lib.mtoken.model.response.OperationListResponse;
+import io.getlime.security.powerauth.rest.api.spring.service.HttpCustomizationService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+/**
+ * Test for {@link MobileTokenService}.
+ *
+ * @author Jan Dusil, jan.dusil@wultra.com
+ */
+@ExtendWith(MockitoExtension.class)
+@ActiveProfiles("test")
+class MobileTokenServiceTest {
+
+    @Mock
+    private PowerAuthClient powerAuthClient;
+
+    @Mock
+    private MobileTokenConverter mobileTokenConverter;
+    @Mock
+    private OperationTemplateService operationTemplateService;
+
+    @Mock
+    private HttpCustomizationService httpCustomizationService;
+
+    @InjectMocks
+    private MobileTokenService tested;
+
+    @Mock
+    private OperationTemplateRepository templateRepository;
+
+    private static final int OPERATION_LIST_LIMIT = 100;
+
+    @Test
+    void testOperationListForUser() throws PowerAuthClientException, MobileTokenConfigurationException {
```

Comment:
Nitpick: Personally, I do not care about particular exceptions in tests and have no problem with a general `Exception`.
```suggestion
    void testOperationListForUser() throws Exception {
```